docker-compose.override.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  express-server:
+    image: ${DOCKERHUB_NAME}/${IMAGE_NAME}:${IMAGE_VERSION}
```

Comment:
Don't need `image` defined here since it's defined in `docker-compose.yml` and inherits in this file